styles.css

```diff
@@ -526,3 +533,177 @@
   justify-content: center;
   align-self: center;
 }
+
+.block {
+  display: none;
+}
+
+@media only screen and (min-width: 420px) {
+  .greetings {
+    font-size: 56px;
+  }
+
+  .title {
+    font-size: 24px;
+  }
+
+  .paragraph {
+    font-size: 16px;
+  }
+
+  .icon {
+    font-size: 22px;
+  }
+
+  .contact-header {
+    font-size: 32px;
+    line-height: 44px;
+  }
+}
+
+@media only screen and (min-width: 650px) {
+  .none {
+    display: none;
+  }
+
+  .project-flex {
+    flex-direction: row;
+    flex-wrap: wrap;
+    width: 100%;
+  }
+
+  .paragraph {
+    max-width: 500px;
+  }
+
+  .work-container {
+    width: 71%;
+    padding-left: 20px;
+  }
+
+  .project-container {
+    width: 49%;
+  }
+
+  .works {
+    flex-direction: row;
+    width: 100%;
+    align-content: space-between;
+  }
+
+  .header-flex {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .nav-list {
+    display: flex;
+    width: 60%;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .img {
+    background-position: right bottom -30%;
+    background-size: 180px;
+  }
+
+  .icon {
+    font-size: 25px;
+  }
+
+  .contact {
+    padding: 120px 80px 0;
+  }
+
+  .ul-list:hover {
+    font-size: 20px;
+    transition: 0.8s ease-out;
+    color: #fff;
+  }
+
+  .img-works {
+    height: auto;
+    width: 60%;
+  }
+}
+
+@media only screen and (min-width: 970px) {
```

Comment:
- [ ] OPTIONAL: This min-width value should serve minimum width for desktop version and better to make it in the range 992px-1100px. Yours is a bit smaller than standard desktop widths.